Clustered execution is supported (#682)
grafana/k6-docs: src/data/markdown/translated-guides/en/07 Misc/06 Archive.md

trigger an implicit creation of a k6 archive that is uploaded and distributed to k6 cloud
load generators for execution.

### Distributed Execution 

[k6-operator](https://github.com/grafana/k6-operator#multi-file-tests) can distribute a k6 test across a Kubernetes cluster. 

> When having a k6 test with multiple files, you can use the archive functionality to bundle the k6 test in a single "archived" file and pass this file to run the test.

Reviewer: ```suggestion
When a k6 test has multiple files, you can use the archive functionality to bundle the k6 test in a single *archived* file and pass this file to run the test.
```
